frontend/public/components/utils/_code-pre.scss

```
@@ -0,0 +1,27 @@
+.co-code {
+  background-color: var(--pf-global--palette--blue-50);
+  color: var(--pf-global--palette--light-blue-600);
+  padding: 3px var(--pf-global--spacer--xs);
+
+  .pf-theme-dark & {
+    background-color: var(--pf-global--palette--blue-700);
+    color: var(--pf-global--palette--light-blue-100);
+  }
```

You could define this block like this to avoid increasing specificity and redefining properties.

```suggestion
  --co-code--BackgroundColor: var(--pf-global--palette--blue-50);
  --co-code--Color: var(--pf-global--palette--light-blue-600);

  background-color: var(--co-code--BackgroundColor);
  color: var(--co-code--Color);
  padding: 3px var(--pf-global--spacer--xs);

  :where(.pf-theme-dark) & {
    --co-code--BackgroundColor: var(--pf-global--palette--blue-700);
    --co-code--Color: var(--pf-global--palette--light-blue-100);
  }
```

---

Switched code